### PR TITLE
Deprecated rules fix

### LINKED ISSUE #14: Deprecated rules
## Describe the bug

After executing the fix command, I see errors about outdated rules

### Expected behaviour

No errors.

### Actual behaviour

```sh
- Rule "compact_nullable_typehint" is deprecated. Use "compact_nullable_type_declaration" instead.
- Rule "function_typehint_space" is deprecated. Use "type_declaration_spaces" instead.
- Rule "native_function_type_declaration_casing" is deprecated. Use "native_type_declaration_casing" instead.
- Rule "single_blank_line_before_namespace" is deprecated. Use "blank_lines_before_namespace" instead.
```

## Additional context

Replace deprecated rules.

### PR DESCRIPTION
**Pull request by helloween141**
## Description

Outdated rules have been replaced with current ones

Fixes #14 (issue)

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I wrote unit tests for my code _(if tests is required for my changes)_
- [x] I have made changes in `CHANGELOG.md` file

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,14 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog][keepachangelog] and this project adheres to [Semantic Versioning][semver].
 
+## Unreleased
+
+### Fixed
```
Comment (eldario) [inline CHANGELOG.md:9]:
```suggestion
### Changed
```

**Comment by helloween141** [comment]
1. CHANGELOG.md spaces syntax fix
2. `friendsofphp/php-cs-fixer` has been updated to `^ 3.32.0`

**Comment by gomzyakov** [comment]
Please, look at https://github.com/avto-dev/php-cs-fixer/pull/17

**Comment by eldario** [comment]
@helloween141 

Resolved in another PR #20 #17

**Pull request closed without merging**